airframe-rx/src/main/scala/wvlet/airframe/rx/RxVar.scala

```diff
@@ -19,7 +19,7 @@ import scala.language.higherKinds
 /**
   * A reactive variable supporting update and propagation of the updated value to the chained operators
   */
-class RxVar[A](protected var currentValue: A) extends Rx[A] with RxVarOps[A] {
+class RxVar[A](private var currentValue: A) extends Rx[A] with RxVarOps[A] {
```

Comment:
Not sure the exact cause of the compilation error, but I guess airframe-surface or DI might have tried to create an accessor function to the protected variable, and caused the compilation error.